wsb-jpa1/src/main/java/com/jpacourse/persistence/dao/impl/PatientDaoImpl.java

```diff
@@ -31,4 +33,31 @@ public VisitEntity createVisit(Long patientId, Long doctorId, LocalDateTime time
 
         return visit;
     }
+
+    @Override
+    public List<PatientEntity> findByLastName(String lastName) {
+        TypedQuery<PatientEntity> query = em.createQuery(
```

Comment:
Na zajęciach prowadzący powiedział że tworzenie nowego EntityManager'a to był błąd ponieważ istnieje on już w AbstractDao które rozszerzamy. Można też zamienić w rozwiązaniu wyżej robionego w ramach lab2 i usunąć deklaracje em w tej klasie
```suggestion
        TypedQuery<PatientEntity> query = entityManager.createQuery(
```